### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by timmc**
Also not sure if the "not it's history" link is supposed to go to your home page.

(I kind of liked "build lawyers between the AGPL 3 code and proprietary code" though...)

### CODE REVIEW

File: _posts/2019-09-09-Shake-Up.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ The no-modification rule is apparent from the text, to those vanishingly few sou
...
 
-I catch myself falling back on an expeident, feel-good notion of "nondiscrimination", too.  But it's not consistent with open source history.  If "open source" includes the preexisting free software copyleft licenses, and not just as payment of old political debt or for stepping stones to march over, then open source included explicitly discriminatory licenses.  RMS wrote GPL to discrimiante against propreitary takers.  If some sofwtare companies used the GPLs to poke other software companies in the eye for closing soruce code, so much the better.
+I catch myself falling back on an expedient, feel-good notion of "nondiscrimination", too.  But it's not consistent with open source history.  If "open source" includes the preexisting free software copyleft licenses, and not just as payment of old political debt or for stepping stones to march over, then open source included explicitly discriminatory licenses.  RMS wrote GPL to discriminate against proprietary takers.  If some software companies used the GPLs to poke other software companies in the eye for closing source code, so much the better.
```
Comment (timmc) [inline _posts/2019-09-09-Shake-Up.md]: There were a number of typos in here, and Github won't display the individual changes unless you use the "rich diff" button. :-/

**Comment by kemitchell** [comment]
Well this is embarrassing! Thanks so much, @timmc!

**Comment by kemitchell** [comment]
Deploying now.

**Comment by kemitchell** [comment]
Linked fixed in 398af3821e46d7384fc1f0b04d061d1d2f4f5477.

**Comment by timmc** [comment]
Haha, it happens! I'm a beta reader for a couple of authors, and there are occasionally these paragraphs that are like 10% typos by mass. I think it happens when people get into a groove. :-)

**Pull request merged**